tests/PySys/environments/environment_c8y.py

```diff
@@ -0,0 +1,117 @@
+import pysys
+from pysys.basetest import BaseTest
+
+"""
+Environment to manage automated connect and disconnect to c8y
+
+Tests that derive from class EnvironmentC8y use automated connect and
+disconnect to Cumulocity. Additional checks are made for the status of
+service mosquitto and service tedge-mapper.
+"""
+
+
+class EnvironmentC8y(BaseTest):
+    def setup(self):
+
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+        self.systemctl = "/usr/bin/systemctl"
+        self.log.info("EnvironmentC8y Setup")
+        self.addCleanupFunction(self.mycleanup)
+
+        # Check if tedge-mapper is in active
+        serv_mapper = self.startProcess(
+            command=self.systemctl,
+            arguments=["status", "tedge-mapper"],
+            stdouterr="serv_mapper1",
+            expectedExitStatus="==3", # 3: disabled
+        )
+
+        if serv_mapper.exitStatus != 3:
```

Comment:
If the `exitStatus` is not 3, then the previous `startProcess` statement itself will fail, right? Making this if condition redundant and unreachable.

Reply:
Absolutely! I somehow expected that a wrong exit code does not abort the test run. I probably never looked if the right messages are printed. I removed the redundant parts.